src/MediaCodecCryptoInfoPattern.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ *      Copyright (C) 2022 Team XBMC
```

Comment:
should be changed with the actual
Copyright (C) 2022 Team Kodi
https://kodi.tv